apps/contracts/contracts/storage/ZengoStorage.sol

```diff
@@ -19,4 +19,13 @@ contract ZengoStorage {
     mapping(address => Structs.Moderator) moderatorStruct;
 
     mapping(address => uint256) public votingPoints;
+
+    function getModeratorInfo(address _moderator) public view returns (Structs.Moderator memory) {
+        require(moderators[_moderator], "Given address is not a Moderator");
+        return moderatorStruct[_moderator];
+    }
+
+    function getModeratorsList() public view returns (address[] memory) {
+        return moderatorList;
+    }
```

Comment:
This will only return the addresses, right? In the frontend we need the info stored in the Struct

Reply:
yes there is a different function for returning the Moderator Struct

Reply:
> yes there is a different function for returning the Moderator Struct

which one? I couldn't found any returning all the mods info :disappointed:

Reply:
it's right about this function